environments/staging-phase-two/group_vars/all/vars.yml

```diff
@@ -45,7 +45,7 @@ nfs_server_for_specials: staging-files00.cnx.org
 
 nfs_connections: "{{ vault_nfs_connections }}"
 
-webview_version: v0.21.0
```

Comment:
O... this is already correct. 👍 

We could have potentially used latest-stable (latest tagged version), but since we are using a development build in production, we can't do that. 😖